core/templates/dev/head/directives/CustomPopoverDirective.js

```diff
@@ -0,0 +1,58 @@
+// Copyright 2019 The Oppia Authors. All Rights Reserved.
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//      http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS-IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+/**
+ * @fileoverview CustomPopover Directive (not associated with reusable
+ * components.)
+ * NB: Reusable component directives should go in the components/ folder.
+ */
+
+// A popover that is shown when its label is hovered or clicked upon, and
+// disappears when focus moves away from its label.
+oppia.directive('customPopover', [
```

Comment:
Is this directive used anywhere? I can't find it used in the code.

Reply:
Yes, I checked too. This is not anywhere in code. Can we remove this then?

Reply:
Yep, you should remove anything that's not used in the code. (Btw, when checking for the use of directives, make sure to convert the name to kebab-case first.)

Reply:
I checked using kebab-case for the directive and it is not used in the code, so I removed this.